src/main/java/com/endava/internship/collections/StudentMap.java

```diff
@@ -1,72 +1,396 @@
 package com.endava.internship.collections;
 
-import java.util.Collection;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
+import java.util.function.Consumer;
 
 public class StudentMap implements Map<Student, Integer> {
+
+    private final int DEFAULT_CAPACITY = 1 << 4;
```

Comment:
I don't like the literal here. Seems I know from where you take it, but please use simple numbers, and no one is going to blame you if you will change to numbers that you like

Reply:
fixed